doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -273,7 +273,7 @@ recovery since the existing metadata pool would not be modified.
 
       cephfs-journal-tool --rank cephfs_recovery:0 journal reset --force --yes-i-really-really-mean-it
 
-#. Now perform the recovery of the metadata pool from the data pool:
+#. Use the following commands to recover the metadata pool from the data pool:
```

Comment:
Weakly suggest s/recover/rebuild/ or even just /build/

Reply:
I agree with you.